plugin/evm/atomic_trie_repair.go

```diff
@@ -79,7 +80,7 @@ func (a *atomicTrie) repairAtomicTrie(bonusBlockIDs map[uint64]ids.ID, bonusBloc
 		}
 	}
 
-	if err := a.metadataDB.Put(repairedKey, []byte{1}); err != nil {
+	if database.PutBool(a.metadataDB, repairedKey, true); err != nil {
```

Comment:
Is there a helper that could be used for `done, err := a.metadataDB.Has(repairedKey)` too?

Reply:
we could use GetBool but then special case database.ErrNotFound , so using Has seemed cleaner to me, but I can change if you prefer

Reply:
I think `Has` is fine, you could comment above this method that putting `true/false` here both are considered repaired.